packages/components/src/components/ScalarCopyButton/ScalarCopyButton.vue

```diff
@@ -0,0 +1,101 @@
+<script setup lang="ts">
+import { ScalarIconCheck, ScalarIconCopy } from '@scalar/icons'
+import { cva, cx } from '@scalar/use-hooks/useBindCx'
+import { useClipboard } from '@scalar/use-hooks/useClipboard'
+import { computed, ref } from 'vue'
+
+import { ScalarButton } from '../ScalarButton'
+
+const props = withDefaults(
+  defineProps<{
+    /** Content to copy to clipboard */
+    content: string | object
+    /** Optional aria label for accessibility */
+    ariaLabel?: string
```

Comment:
Elsewhere in the component library we just use `label` or `srLabel`. I've stopped writing `ariaLabel` in camelcase because then people use it on actual HTML elements and it doesn't work (it needs to be `aria-label`)